code/src/pages/cart/CartPage.scss

```diff
@@ -0,0 +1,78 @@
+@import "@design-system";
+
+.spa-cart-page {
+  padding: spacing(10);
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    gap: spacing(5);
+  }
+
+  &__items,
+  &__order-summary {
+    padding: spacing(5);
+    border: 1px solid var(--spa-order-summary-border-color, $white);
+    border-radius: spacing(1);
+    background-color: var(--spa-order-summary-background-color, $white);
+  }
+
+  &__items {
+    width: 60%;

```

Comment:
Avoid to use % with width. It will be hard to debug and may create a lot of bugs in the future

Reply:
> Avoid to use % with width. It will be hard to debug and may create a lot of bugs in the future

Explain please. Hard to debug what ? width ?

Reply:
![image](https://github.com/idx-academy/spa-orders/assets/118522791/4d44c811-6493-40c0-b5a8-a5c16790308d)